src/Shell/Command/GitDriverException.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\CloudPatches\Shell\Command;
+
+use Symfony\Component\Process\Exception\ProcessFailedException;
+use Throwable;
+
+/**
+ * Git patch driver exception
+ */
+class GitDriverException extends DriverException
```

Comment:
GitDriverException and PatchDriverException has the same implementation 
Do we need them both? Or just use DriverException?